python/lsst/ts/dsm/dsm_csc.py

```diff
@@ -176,26 +176,26 @@ def process_event(self, event):
 
         Parameters
         ----------
-        event : `aionotify.Event`
+        event : `asyncinotify.Event`
             Payload containing the I/O event information.
         """
-        self.log.debug(f"Event: Flags = {event.flags}, Name = {event.name}")
-        if event.flags & aionotify.Flags.CLOSE_WRITE:
-            if event.name.endswith("yaml"):
-                self.process_yaml_file(event.name)
-            if event.name.endswith("dat"):
-                self.process_dat_file(event.name)
+        self.log.debug(f"Event: Mask = {event.mask}, Name = {event.name}")
+        if event.mask & asyncinotify.Mask.CLOSE_WRITE:
```

Comment:
Is this test necessary, since you set the mask when you create the notifier?

If this should always be true, then please consider logging a message when the test fails. I admit that could be annoying if it fails frequently, but it might be informative if failures are rare.

Reply:
I think this was a hold over from the other package. I've made the code on this more like the asyncinotify package works and it seems to do just fine.